### PR TITLE
Document new changes

### PR DESCRIPTION
**Pull request by slimit75**
This was an attempt to document changes to the config that I am making in a separate pull request to [Kristify](https://github.com/Kristify/Kristify) and therefore won't be ready until that pull request is done. The full changes are below:

- [x] Document new options
- [x] Fix typos & table formatting that my IDE was complaining about
- [x] Add new options to configgen

### CODE REVIEW

**Review by piprett — APPROVED** [review]
A lot of great work here, and thanks for fixing all the typos! Ill test the new code added more in-depth later, but I am happy with all the markdowns!

File: docs/configgen/script.js [hunk 1]
```diff
@@ -72,7 +82,19 @@ ${this.getStorage()}
...
   },
-  speakers = {}
+  speakers = {},
+  shopSync = {
+    enabled = ${this.ssEnabled},
+    modem = "${this.ssModem}",
+    owner = "${this.ssOwner}",
+    multiShop = ${this.ssMultiShop || "nil"},
+    location = {
+      broadcastLocation = ${this.ssLocEnabled},
```
Comment (piprett) [inline docs/configgen/script.js]:
```suggestion
      broadcastLocation = ${this.ssLocEnabled || 'false'},
```

File: docs/configgen/index.html [hunk 2]
```diff
@@ -172,6 +172,71 @@ <h3 class="pb-3">Webhooks</h3>
...
+              </div>
+
+              <div class="form-floating mb-3">
+                <input type="text" class="form-control rounded-3" id="ssMultiShop" v-model="ssMultiShop"/>
+                <label for="ssMultiShop">multiShop ID</label>
+                <div class="form-text pt-0 pb-3">Integer ID of the shop, counting from 1. Only applicable if multiple shops are running on the same computer. <b>If only one shop is running on the computer, leave this value empty!!</b></div>
+              </div>
+
+              <div class="form-floating mb-3">
+                <select id="ssLocEnabled" class="form-select" v-model="ssLocEnabled" required>
```
Comment (piprett) [inline docs/configgen/index.html]:
```suggestion
                <select id="ssLocEnabled" class="form-select" v-model="ssLocEnabled">
```

**Review by piprett — CHANGES_REQUESTED** [review]
It appears that the "broadcastLocation" option is required to be specified, even if ShopSync is disabled. I've pointed it out in some specific review comments, along with a fix.

**Comment by slimit75** [comment]
My bad, should be fixed now

**Review by piprett — APPROVED** [review]

**Pull request merged**